.github/workflows/pr-test.yml

```diff
@@ -4,19 +4,13 @@ on:
   push:
     branches: [ main ]
     paths:
-      - "python/pyproject.toml"
-      - "python/sglang/**"
+      - "python/**"
       - "test/**"
-      - "docs/**"
-      - "scripts/**"
```

Comment:
nit: maybe we need to keep it, because e.g. when scripts/ci_install_dependency.sh changes, we may need to re-run all pr-test to ensure that change does not break things.

(same for vllm-dependency-test.yml etc)

Reply:
fixed